projects/mci/inc/regen_braking.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <stdbool.h>
+
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "status.h"
+
+typedef enum { REGEN_DISABLED = 0, REGEN_ENABLED = 1 } RegenBrakingState;
```

Comment:
Ah, centre console just uses a `bool` to represent "is regen braking enabled" - probably best to just use `bool` for consistency.

Reply:
Done, changed to bool